proto/axelar/multisig/v1beta1/types.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+package axelar.multisig.v1beta1;
+
+option go_package = "github.com/axelarnetwork/axelar-core/x/multisig/types";
+
+import "gogoproto/gogo.proto";
+import "axelar/utils/v1beta1/threshold.proto";
+import "axelar/snapshot/exported/v1beta1/types.proto";
+import "axelar/multisig/exported/v1beta1/types.proto";
+
+message Key {
+  option (gogoproto.stable_marshaler) = true;
+
+  string id = 1 [
+    (gogoproto.customname) = "ID",
+    (gogoproto.casttype) =
+        "github.com/axelarnetwork/axelar-core/x/multisig/exported.KeyID"
+  ];
+  snapshot.exported.v1beta1.Snapshot snapshot = 2
+      [ (gogoproto.nullable) = false ];
+  map<string, bytes> pub_keys = 3 [ (gogoproto.castvalue) = "PublicKey" ];
```

Comment:
```suggestion
  map<string, bytes> pub_keys = 3 [ (gogoproto.castvalue) = "PublicKeys" ];
```

Reply:
Right now, it generates `map[string]PublicKey`. Do you wanna create a alias type for the entire map instead?